modules/ROOT/pages/start-using-sdk.adoc

```diff
@@ -17,9 +17,11 @@ Please see xref:compatibility-versions-features.adoc#platform-compatibility[the
 == Requirements
 
 First, install libcouchbase following the instructions xref:2.10@c-sdk::sdk-release-notes.adoc[here].
+Note that the Windows installation handles this automatically.
 
-The Couchbase Python SDK 2.5 will run any any version of Python above 2.7 or 3.5 (Python 3.x only on Windows).
-Other pre-requisites will depend upon your Operating System platform.
+The Couchbase Python SDK 2.5 will run any any version of Python above 2.7 or 3.5 
+(Python 3.x only on Windows -- with 3.6 and above _strongly_ recommended, in order to ensure libcouchbase is automatically installed via Python wheel).
```

Comment:
(Python 3.x only on Windows -- with 3.6 and above _strongly_ recommended, in order to ensure libcouchbase and the compiled C bindings are automatically installed via Python wheel, rather than requiring manual installation/compilation)